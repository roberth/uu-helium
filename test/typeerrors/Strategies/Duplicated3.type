
       x :: a ; y :: b ;
  -------------------------
        x +++ y :: Int;;

constraints x
a == Int : "linkerkant moet een Int zijn"
constraints x
b == Int : "rechterkant moet een Int zijn";
