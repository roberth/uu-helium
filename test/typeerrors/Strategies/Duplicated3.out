Compiling typeerrors/Strategies/Duplicated3.hs
The constraints for meta-variable "x" of type strategy rule "x +++ y" can only be inserted at one place
